Gemfile.lock

```diff
@@ -69,48 +70,50 @@ GEM
     ast (2.3.0)
     attr_encrypted (3.0.3)
       encryptor (~> 3.0.0)
-    awrence (0.1.0)
-    aws-sdk (2.6.12)
-      aws-sdk-resources (= 2.6.12)
-    aws-sdk-core (2.6.12)
+    awrence (1.0.0)
+    aws-sdk (2.10.21)
+      aws-sdk-resources (= 2.10.21)
+    aws-sdk-core (2.10.21)
+      aws-sigv4 (~> 1.0)
       jmespath (~> 1.0)
-    aws-sdk-resources (2.6.12)
-      aws-sdk-core (= 2.6.12)
+    aws-sdk-resources (2.10.21)
+      aws-sdk-core (= 2.10.21)
+    aws-sigv4 (1.0.1)
     axiom-types (0.1.1)
       descendants_tracker (~> 0.0.4)
       ice_nine (~> 0.11.0)
       thread_safe (~> 0.3, >= 0.3.1)
     binding_of_caller (0.7.2)
       debug_inspector (>= 0.0.1)
-    brakeman (3.4.1)
+    brakeman (3.7.0)
     breakers (0.2.3)
       faraday (>= 0.7.4, < 0.10)
       multi_json (~> 1.0)
     builder (3.2.3)
-    bundler-audit (0.5.0)
+    bundler-audit (0.6.0)
       bundler (~> 1.2)
       thor (~> 0.18)
-    byebug (8.2.2)
+    byebug (9.0.6)
     carrierwave (0.11.2)
       activemodel (>= 3.2.0)
       activesupport (>= 3.2.0)
       json (>= 1.7)
       mime-types (>= 1.16)
       mimemagic (>= 0.3.0)
-    carrierwave-aws (1.0.2)
-      aws-sdk (~> 2.0)
+    carrierwave-aws (1.2.0)
+      aws-sdk (~> 2.1)
       carrierwave (>= 0.7, < 2.0)
     case_transform (0.2)
       activesupport
-    childprocess (0.5.9)
+    childprocess (0.7.1)
       ffi (~> 1.0, >= 1.0.11)
     clam_scan (0.0.2)
-    climate_control (0.1.0)
+    climate_control (0.2.0)
     cliver (0.3.2)
     coderay (1.1.1)
     coercible (1.0.0)
       descendants_tracker (~> 0.0.1)
-    combine_pdf (0.2.36)
+    combine_pdf (1.0.6)
```

Comment:
The Datestamp task uses this, which tests the final output to make sure the text is present, iirc.